packages/next/components/postDetail/biasedVotingItem.js

```diff
@@ -22,13 +22,22 @@ const LabelWrapper = styled.div`
   }
 `;
 
-function BiasedVotingItem({ label, value, space }) {
+function BiasedVotingItem({ label = "", value, space }) {
+  const Label = <div className="label">{label}</div>;
```

Comment:
we don't capitalize  regular variables ,  only contants & tags shall be capitalized.
in this case, `Label` is a element rather than a tag name

maybe name it `displayLabel` ?

Reply:
in a minute